vimmo/db/weekly_update.py

```diff
@@ -38,14 +41,36 @@ def fetch_latest_versions(api_url):
         # Move to the next page if available
         api_url = data.get("next")
 
-    logging.info(f"Fetched {len(latest_versions)} panel versions.")
+    logging.info(f"Fetched {len(latest_versions)} panels from PanelApp API.")
     return latest_versions
 
+def fetch_latest_version_genes(id, latest_version):
+    latest_genes = []
+
+    genes_url = f"https://panelapp.genomicsengland.co.uk/api/v1/panels/{id}/?format=json&version={latest_version}"
```

Comment:
Cant remember if we discussed, just be careful that this is only looking at latest signed off panels endpoint instead of latest panel endpoint.

Reply:
Yeah all good, this function is called in `update_or_insert_panel_versions()` with the latest versions coming from the latest signed off versions endpoint , provided as argument to `fetch_latest_versions()` in `main()` (line 215)